asmcomp/i386/proc.ml

```diff
@@ -107,7 +107,7 @@ let word_addressed = false
    These globals are denoted by Incoming and Outgoing stack locations
    with negative offsets, starting at -64.
    Unlike arguments passed on stack, arguments passed in globals
-   do not prevent tail-call elimination.  The caller stores arguments
+   do not prevent tailcall elimination.  The caller stores arguments
```

Comment:
Why replace tail-call by the rarer form tailcall? (see for instance https://en.wikipedia.org/wiki/Tail_call)